sw/airborne/modules/computer_vision/opticflow/opticflow_calculator.h

```diff
@@ -49,6 +49,7 @@ struct opticflow_t {
   uint16_t window_size;             ///< Window size for the blockmatching algorithm (general value for all methods)
   uint16_t search_distance;         ///< Search distance for blockmatching alg.
   uint8_t derotation;             ///< Derotation switched on or off (depended on the quality of the gyroscope measurement)
+  uint8_t median_filter;          ///< Decides to use a median filter on the velocity
```

Comment:
If you only use this to enable/disable the median_filter, it should be a `bool`.
Same actually goes for `derotation`